src/mmrelay/db_utils.py

```
@@ -715,13 +874,625 @@ def _store(cursor: sqlite3.Cursor) -> None:
+    select_by_id_sql = _SELECT_NAME_VALUE_BY_ID_SQL_BY_TABLE.get(table)
+    if column_name is None or select_sql is None or select_by_id_sql is None:
+        raise _InvalidNamesTableError(table)
+
+    manager = _get_db_manager()
+    sorted_ids = sorted(current_ids)
+    supports_json_each = True
+    supports_json_each_fn = type(manager).__dict__.get("supports_json_each")
+    if callable(supports_json_each_fn):
+        supports_json_each = bool(supports_json_each_fn(manager))
+    elif "_supports_json_each" in getattr(manager, "__dict__", {}):
+        supports_json_each = bool(getattr(manager, "_supports_json_each"))
```

Ruff (0.15.6)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)